redbot/cogs/mod/settings.py

```diff
@@ -106,6 +112,24 @@ async def mentionspam(self, ctx: commands.Context):
         Manage the automoderation settings for mentionspam.
         """
 
+    @mentionspam.command(name="strict")
+    @commands.guild_only()
+    async def mentionspam_strict(self, ctx: commands.Context, toggle: bool):
```

Comment:
The other commands in this cog are toggles, not passed booleans. You should follow that pattern.